keyboards/planck/rev7/matrix.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright 2023 Jack Humbert <jack.humb@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "config.h"
```

Comment:
shouldn't need.  And should be converted to matrix lite.
```suggestion
```

Reply:
I can convert this & rev6_drop's matrix to lite separately.

Reply:
> I can convert this & rev6_drop's matrix to lite separately.

"separately" as in not as part of this PR? If so, we would usually block merge on such a decision.

Reply:
Separately in another PR, because I don't want to update rev6_drop here and would rather have their implementations match for the time being.

Reply:
I agree `rev6_drop` should not be changed as part of this PR. However the matrix code is not the same, so it being aligned is already false. Its called out in the PR checklist, and has been requested/required on many previous PRs.

Reply:
It should be the same matrix code aside from the encoder matrix additions.

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/16520359/acc2661f-00a4-40f9-b645-ed8d8912c413)

Nope, different, not really close given one is using iteration and GPIO abstraction and the other ChibiOS directly.